apps/constants/base.py

```diff
@@ -510,6 +510,8 @@
     {'name': _('Level 9'), 'points': 3000000},
 ]
 
+REVIEW_LIMITED_DELAY_HOURS = 20
```

Comment:
Would you want to change this frequently? If so, it might be better as a setting.